kong/db/schema/init.lua

```diff
@@ -1201,6 +1201,7 @@ local function run_entity_check(self, name, input, arg, full_check, errors)
     local value = get_field(input, fname)
     if value == nil then
       if (not checker.run_with_missing_fields) and
+         (not arg.run_with_missing_fields) and
```

Comment:
This change should have it's own tests.

Reply:
Test case added, Do you think it is enought?

Reply:
Looks good, thanks for adding!